sqlutil/sqlutil.go

```diff
@@ -73,3 +73,99 @@ func (l *StringList) Scan(v interface{}) error {
 	*l = strs
 	return nil
 }
+
+// Bool add the capability to the sql driver to work directly with bool types.
+type Bool bool
+
+// Scan converts the different types of representation of a boolean in the
+// database into a bool type.
+func (b *Bool) Scan(src interface{}) error {
+	if b == nil {
+		return fmt.Errorf("boolean not initialized")
+	}
+
+	switch v := src.(type) {
+	case int64:
+		*b = v != 0
+	case float64:
+		*b = v != 0
```

Comment:
Can `src` also be one of the other `int` and `uint` types or `float32`? Or is it always `int64` and `float64`?

Reply:
According to the documentation of [Scanner interface](https://golang.org/pkg/database/sql/#Scanner) this method can receive only the following types:

* int64
* float64
* bool
* []byte
* string
* time.Time
* nil